shuffle/cli/src/test.rs

```diff
@@ -225,21 +226,36 @@ pub enum TestCommand {
     All {
         #[structopt(short, long)]
         project_path: Option<PathBuf>,
+
+        #[structopt(short, long)]
+        network: Option<String>,
     },
 }
 
-pub fn handle(cmd: TestCommand) -> Result<()> {
+pub fn handle(cmd: TestCommand, home: &Home) -> Result<()> {
```

Comment:
I like these series of changes to `fn handle`. I personally think we should have `home: &Home` be the first parameter rather than the last, to show consistency w the other handle methods.

Reply:
will change method headers to intake home: &Home first!